addons/stock/views/stock_quant_views.xml

```diff
@@ -180,13 +180,21 @@
         <field name="context">{'search_default_internal_loc': 1, 'search_default_productgroup':1, 'search_default_locationgroup':1}</field>
         <field name="res_model">stock.quant</field>
     </record>
+
     <record model="ir.actions.act_window" id="location_open_quants"> <!-- Used in location -->
         <field name="context">{'search_default_productgroup': 1}</field>
         <field name="domain">[('location_id', 'child_of', active_ids)]</field>
         <field name="name">Current Stock</field>
         <field name="res_model">stock.quant</field>
     </record>
 
+    <record id="location_open_quants_tree_view" model="ir.actions.act_window.view">
+        <field name="sequence" eval="1"/>
+        <field name="view_mode">tree</field>
+        <field name="view_id" ref="view_stock_quant_tree"/>
```

Comment:
No need for a new action, you can add the `view_id` directly to the `location_open_quants` action :wink: